fml/platform/android/jni_util.cc

```diff
@@ -142,6 +142,25 @@ ScopedJavaLocalRef<jobjectArray> VectorToStringArray(
   return ScopedJavaLocalRef<jobjectArray>(env, joa);
 }
 
+ScopedJavaLocalRef<jobjectArray> VectorToBufferArray(
+    JNIEnv* env,
+    const std::vector<std::vector<uint8_t>>& vector) {
+  FML_DCHECK(env);
+  ScopedJavaLocalRef<jclass> byte_buffer_clazz(
+      env, env->FindClass("java/nio/ByteBuffer"));
+  FML_DCHECK(!byte_buffer_clazz.is_null());
+  jobjectArray joa =
+      env->NewObjectArray(vector.size(), byte_buffer_clazz.obj(), NULL);
+  ASSERT_NO_EXCEPTION();
+  for (size_t i = 0; i < vector.size(); ++i) {
+    ScopedJavaLocalRef<jobject> item(
+        env,
+        env->NewDirectByteBuffer((void*)(vector[i].data()), vector[i].size()));
+    env->SetObjectArrayElement(joa, i, item.obj());
```

Comment:
I'm confused about how this works. Doesn't the ScopedJavaLocalRef delete the obj in its dtor?

Reply:
I copied over from the String converting method

Reply:
Ok. I'm probably just misunderstanding how `JNIEnv::DeleteLocalRef` works.